platform/android/java/lib/src/org/godotengine/godot/GodotView.java

```diff
@@ -156,24 +164,49 @@ private void init(XRMode xrMode, boolean translucent, int depth, int stencil) {
 				 */
 
 				if (GLUtils.use_32) {
-					setEGLConfigChooser(translucent
+					eglConfigChooser = translucent
 									? new RegularFallbackConfigChooser(8, 8, 8, 8, 24, stencil,
 											  new RegularConfigChooser(8, 8, 8, 8, 16, stencil))
 									: new RegularFallbackConfigChooser(8, 8, 8, 8, 24, stencil,
```

Comment:
clang-format and ternary operators, fun... now that it's no longer a function call it can be aligned with two tabs as configured, no longer four tabs as some arbitrary approximation of the starting parenthesis.

Reply:
Amended!